scripts/cleanup-dev-resources.sh

```
@@ -0,0 +1,102 @@
+    echo "🗑️  다음 인스턴스들을 종료합니다: $INSTANCE_IDS"
+    aws ec2 terminate-instances --instance-ids $INSTANCE_IDS
+    echo "⏳ 인스턴스 종료 완료 대기 중..."
+    aws ec2 wait instance-terminated --instance-ids $INSTANCE_IDS
+    echo "✅ EC2 인스턴스 정리 완료"
+fi
+
+# AutoCleanup 태그가 있는 보안 그룹 정리 (단, 인스턴스가 사용하지 않는 것만)
+echo "🔍 미사용 보안 그룹 검색 중..."
+SECURITY_GROUPS=$(aws ec2 describe-security-groups \
+    --filters "Name=tag:AutoCleanup,Values=true" \
+    --query 'SecurityGroups[?length(ReferencingSecurityGroupIds)==`0`].GroupId' \
```

Shellcheck (0.10.0)

[info] Expressions don't expand in single quotes, use double quotes for that.

(SC2016)